src/NzbDrone.Core/Indexers/GetStrike/GetStrikeApi.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Newtonsoft.Json;
+
+namespace NzbDrone.Core.Indexers.GetStrike
+{
+    public class GetStrikeResponse
+    {
+        [JsonProperty(Required = Required.Always)]
+        public int statuscode { get; set; }
+        [JsonProperty(Required = Required.Always)]
+        public int results { get; set; }
+        [JsonProperty(Required = Required.Always)]
+        public double responsetime { get; set; }
+        public object torrents { get; set; }
+    }
+
+    public class GetStrikeNotFound
+    {
+        [JsonProperty(Required = Required.Always)]
+        public int statuscode { get; set; }
+        [JsonProperty(Required = Required.Always)]
+        public string message { get; set; }
+    }
+
+    public class GetStrikeQueryResponse
+    {
+        [JsonProperty(PropertyName = "torrent_hash")]
```

Comment:
Set the property name to the JSON property name (same as we do for other similar cases).